dev/remsh

```diff
@@ -23,6 +23,12 @@ if [ -z $HOST ]; then
     HOST="127.0.0.1"
 fi
 
+if [ -n "$COOKIE" ]; then
+    _cookie_flag="-setcookie $COOKIE"
+else
+    _cookie_flag=""
```

Comment:
any reason for the else clause when empty is the default?

Reply:
No, there is not any :-)  Thanks for catching this.